src/main/java/de/redstoneworld/redplayerutils/commands/RedHealthCommand.java

```diff
@@ -18,7 +18,7 @@ protected String getValue(Player player) {
     @Override
     protected boolean applyValue(Player target, String input) {
         double healthLevel = Double.parseDouble(input);
-        if (healthLevel > 20) {
+        if (healthLevel > 20 || healthLevel < 0) {
```

Comment:
I think I would swap it: `healthLevel < 0 || healthLevel > 20`

Reply:
Ok. I have changed this for all commands.